compiler/rustc_expand/src/mbe/transcribe.rs

```diff
@@ -157,6 +157,8 @@ pub(super) fn transcribe<'a>(
                 // done. Otherwise, we unwind the result_stack to append what we have produced to
                 // any previous results.
                 Frame::Delimited { delim, span, spacing, .. } => {
+                    // njn: hmm
+                    let spacing = DelimSpacing::new(spacing.open, Spacing::Alone);
```

Comment:
Are you sure this is the place where we expand `$(TOKENS)*` and can modify the last token of `TOKENS`?
It doesn't look like that.